src/utils/yarn.js

```diff
@@ -22,6 +22,23 @@ function depTypeToFlag(depType) {
   return flag ? `--${flag}` : flag;
 }
 
+export async function install(cwd: string, pureLockfile?: boolean) {
```

Comment:
This was the only thing I debated, the signature here is a bit weird.

```
install(cwd, pureLockfile)
```

I feel like maybe it should just take a single opts object maybe? It's only internal API so I'm not super worried about it, and am happy to change if you have a preference